### PR TITLE
add revisions functionality

### LINKED ISSUE #16: add revisions functionality
add the view revision history option, which provides dates and time of documents previous revisions

### LINKED ISSUE #40: add drive info functionality
Using the command by either specifying the sharing link or the file id one should be able to view relevant information of the file/folder like size of it, members / owners, mimetype , creation date, last modified date, parent folder etc.
Note: The data displayed regarding the same should be organised and layout should be clean enough to read.

### PR DESCRIPTION
**Pull request by raghav-dalmia**
## History
I have written a function `history()` in `actions.py`, which prints history (that is information about previously used commands). This function has two flags
1) `--date`: User has to pass date as an argument and it will filter out the history.
2) `--clear`: This flag deletes the entire history.

#### Output of `drive history`
-----
Screenshot from 2019-03-13 23-36-42

#### Output of `drive history --clear`
-----
Screenshot from 2019-03-13 23-24-51

#### Output od `drive history --date <date>`
-----
Screenshot from 2019-03-14 00-02-48

## Log
I have written a function `get_revision()` in `actions.py` which take file id/ link as a mandatory argument and print all the revisions related to that file. This function has three flags and all the flags take revision_id as an argument:
1) `--get`: it provides more information about that particular revision
2)`--delete`: this flag deletes the revision
3)`--save`: This flag keep revision forever, even if it is no longer the head revision. If not set, the revision will be automatically purged 30 days after newer content is uploaded.

Screenshot from 2019-03-16 01-03-39
Fixes #40 
Fixes #16

### CODE REVIEW

**Comment by raghav-dalmia** [comment]
@nurdtechie98 done. Sorry for the late reply.

**Comment by raghav-dalmia** [comment]
@nurdtechie98 @inishchith please review this PR. If it requires any changes please let me know.

**Comment by nurdtechie98** [comment]
@raghav-dalmia thanks for the pr, great work. LGTM

**Comment by raghav-dalmia** [comment]
@inishchith @parulagg27 @sauravchirania @shivam1708 please review this PR.

File: drive_cli/utils.py [hunk 1]
```diff
@@ -18,6 +19,46 @@
...
+    return history
+
+
+def save_history(info):
+    date_time = datetime.now().strftime("%d/%m/%Y %H:%M:%S").split(" ")
+    date = date_time[0]
+    time = date_time[1]
+    commands = sys.argv
+    command = commands[1]
+    commands[2:]
```
Comment (inishchith) [inline drive_cli/utils.py]: Not sure why this line is present here.

Comment (raghav-dalmia) [reply drive_cli/utils.py]: it's my mistake, sorry.

**Comment by raghav-dalmia** [comment]
by mistake, I have pushed `file-info` too. Please review it too. Sorry about that.

**Comment by nurdtechie98** [comment]
@inishchith anything more to changed ?

**Comment by inishchith** [comment]
@raghav-dalmia Would appreciate if you could resolve the conflict and update the thread. 
Thanks for  being so persistent :)

@nurdtechie98 Did you test this locally?

**Comment by raghav-dalmia** [comment]
Done @inishchith . Please review asap.

**Comment by nurdtechie98** [comment]
@inishchith yes I did. Seems good to me

**Pull request merged**